### PR TITLE
Linode Graphs Filler

### LINKED ISSUE #306: Flesh out the graphs
We can still use made up data but we should have each of the graphs in place and the logic hooked up to switch between them.

### PR DESCRIPTION
**Pull request by na3d**
Closes #306

### CODE REVIEW

File: src/linodes/layouts/LinodeGeneral.js [hunk 1]
```diff
@@ -25,49 +25,121 @@ export class LinodeGeneral extends Component {
     this.componentDidMount = loadLinode.bind(this);
     this.renderDetails = this.renderDetails.bind(this);
     this.renderGraphs = this.renderGraphs.bind(this);
+    this.graphSelection = this.graphSelection.bind(this);
+    this.graphUpdate = this.graphUpdate.bind(this);
+    this.graphRangeUpdate = this.graphRangeUpdate.bind(this);
+    this.graphSourceUpdate = this.graphSourceUpdate.bind(this);
+    this.graphState = {
```
Comment (eatonphil) [inline src/linodes/layouts/LinodeGeneral.js]: You should be using `this.state` and you should be updating it using `this.setState`. They are magical React functions.

Comment (na3d) [inline src/linodes/layouts/LinodeGeneral.js]: Magic has no place in programming! haha

Comment (na3d) [inline src/linodes/layouts/LinodeGeneral.js]: You the man!  What is this witchcraft?

Comment (SirCmpwn) [inline src/linodes/layouts/LinodeGeneral.js]: +1

File: src/linodes/layouts/LinodeGeneral.js [hunk 2]
```diff
@@ -25,49 +25,120 @@ export class LinodeGeneral extends Component {
...
-    const now = new Date().getTime() / 1000;
-    const data = [
-      {
-        name: 'CPU Usage',
-        values: _.range(now - 86400, now, 30 * 60)
-          .map(ts => ({ x: ts, y: Math.random() * 5 })),
-        strokeWidth: 2,
+  graphRangeUpdate(value) {
+    this.setState({ range: value });
+    this.renderGraphs();
```
Comment (eatonphil) [inline src/linodes/layouts/LinodeGeneral.js]: Doesn't seem like you need this here

File: src/linodes/layouts/LinodeGeneral.js [hunk 3]
```diff
@@ -25,49 +25,120 @@ export class LinodeGeneral extends Component {
...
+        strokeWidth: 2,
+      }],
+      yLabel: source.yLabel,
+      xLabel: source.xLabel,
+      yDomain: source.yDomain,
+      xDomain: d => new Date(d.x),
+    };
+    return data;
+  }
+  renderGraphs() {
```
Comment (eatonphil) [inline src/linodes/layouts/LinodeGeneral.js]: Line please

File: src/linodes/layouts/LinodeGeneral.js (hunk 2 above)
Comment (SirCmpwn) [inline src/linodes/layouts/LinodeGeneral.js]: Yeah these aren't necessary. In this case they're no-ops.

**Comment by SirCmpwn** [comment]
LGTM

**Pull request merged**